django/views/templates/csrf_403.html

```diff
@@ -64,7 +64,7 @@ <h2>Help</h2>
     <code>csrf_protect</code> on any views that use the <code>csrf_token</code>
     template tag, as well as those that accept the POST data.</li>
 
-    <li>The form has a valid CSRF token. After logging in in another browser
+    <li>The form has a valid CSRF token. After logging in another browser
```

Comment:
This is not a typo, please revert.

Reply:
As a native English speaker, I found this sentence hard to digest:
 - the "logging in in" bit
 - multiple clauses
 - "login" mentioned multiple times (especially "after a login" which is repetitive)

I wasn't really sure the best way to rephrase it. I asked ChatGPT to rephrase it, and whilst not 100% correct, it offers some nice suggestions:

> After logging in on another browser tab or navigating back from the login page, you may find it necessary to refresh the form page, as the token undergoes rotation following each login.

Just curious, though, re "hitting the back button after a login":  I'm pretty certain we can safely remove that as I thought browsers have long since moved on from that old double-submit behaviour from the 1990s? 🤔 (ie when following POST-redirect-GET, the browser won't store the POST in the history?)

Reply:
We discussed this once, and decided to leave as it is. I'll try to find PR later.

Reply:
The previous PR where this was discussed is #17109.

Given that this is a recurrent point of confusion, I think we need to re-phrase. If anything, for the sake of lowering the barrier to non native speakers to understand what the docs mean.

Reply:
Oh haha I remember that now.

I thought felix was referring to removing the bit about navigating back after login.

Reply:
Suggestion, it uses more common words and I think it flows better:

After logging in using another browser tab or navigating back after login, you may need to reload the page containing the form, as the token is rotated once logged in.

Reply:
@cuinix Hi! I think you resolved this comment but did not apply any suggestion. Was that intentional?

Reply:
I agree with the previous discussion that it's not worth changing.

Reply:
Thanks @felixxm for restating your thoughts. I understand that you feel it's not worth changing, but I still think we should. This is a recurring pain-point for docs readers as we have witnessed. Let's talk more about it, I'll propose a separated PR.

Reply:
I don't think it is a "pain-point". It is just something that folks find when scanning Django's source code for repeated words, no more no less. We have plenty of bugs, features, and optimization to review. It's not worth wasting time on grammatically correct sentence, there is nothing wrong in it.